istio-auth/cloudbuild-test.yaml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+## This Cloud Build script tests against the Istio authorization rules
+
+steps:
+# Add a step to delete all keys afterwards, delay failure til later.
+- name: 'google/cloud-sdk:slim'
+  id: Create Firebase Service Account Key
+  entrypoint: /bin/bash
+  args:
+  - -c
+  - |
+    FB_ADMIN_SA=$(gcloud iam service-accounts list --filter="displayName=firebase-adminsdk" --format="value(email)")
+    gcloud iam service-accounts keys create sa-key.json --iam-account=$${FB_ADMIN_SA}
```

Comment:
Unfortunately this will not be automatable because service account key export is disabled for Google-owned GCP projects by default, including the project that runs the test infra for this project. Is there another way to do this that doesn't require exporting a service account key? I would assume that the Cloud Build service account could be granted Firebase Admin permissions instead and that would allow this to work.